python/telegram_bot/github_sync.py

```
@@ -0,0 +1,80 @@
+            with open(WATCHLIST_FILE, "r") as f:
+                return json.load(f)
+        return []
+        
+    g = Github(GITHUB_TOKEN)
+    try:
+        repo = g.get_repo(GITHUB_REPO)
+        file_content = repo.get_contents(WATCHLIST_FILE)
+        return json.loads(file_content.decoded_content.decode('utf-8'))
+    except UnknownObjectException:
+        return []
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)